pkgs/development/python-modules/fvcore/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, huggingface-hub
+, iopath
+, numpy
+, pillow
+, pyyaml
+, shapely
+, tabulate
+, termcolor
+, torch
+, torchvision
+, tqdm
+, yacs
+}:
+
+buildPythonPackage rec {
+  pname = "fvcore";
+  disabled = pythonOlder "3.6";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "fvcore";
+    rev = version;
+    sha256 = "1vz9qif5iaxhwks5mhg9lrfr16aj5kb108dgphhcv0iq4nla2wsa";
+  };
+
+  nativeBuildInputs = [
+    iopath
+    numpy
+    pillow
+    pyyaml
+    tabulate
+    termcolor
+    tqdm
+    yacs
+  ];
+
+  propagatedBuildInputs = [
+    torch
+    shapely
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pytestFlagsArray = [
+    "tests"
+  ];
+
+  pythonImportsCheck = [
+    "fvcore"
+    "fvcore.nn"
+  ];
+
+  meta = with lib; {
+    description = "Collection of common code that's shared among different research projects in FAIR computer vision team.";
+    homepage = "https://github.com/facebookresearch/fvcore";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ swaddle ];
```

Comment:
Please add yourself to the list of maintainers in the file `maintainers/maintainer-list.nix`. There are informative comments on how to do so at the start of the file.